apps/E2E/src/Button/specs/Button.spec.win.ts

```diff
@@ -53,24 +69,36 @@ describe('Button Functional Testing', () => {
   });
 
   it('Validate OnClick() callback was fired -> Click', async () => {
```

Comment:
nit this reads a bit backwards to me. I think it should be "condition/action -> expected result" so:

"Click on Button -> Validate OnClick() callback was fired"
"Type "Enter" on Button -> Validate OnClick() callback was fired"
"Type "Space" on Button -> Validate OnClick() callback was fired"

same for ExperimentalButton